sql/variables/status_variables.go

```diff
@@ -138,67 +153,67 @@ var statusVars = map[string]sql.StatusVariable{
 		Name:    "Aborted_clients",
 		Scope:   sql.StatusVariableScope_Global,
 		Type:    types.NewSystemIntType("Aborted_clients", 0, 0, false),
-		Default: int64(0),
+		Default: atomic.Uint64{},
```

Comment:
Better to make default values explicit  (`atomic.Uint64{0}`)

Reply:
we would need to make a default object and load to make the zero explicit, i think
```go
defaultInc := atomic.Uint64{}
defaultInc.Store(0)
...
Default: defaultInc,
```
which seems more difficult?